run-checks

```diff
@@ -378,6 +378,7 @@ if [ "$UNIT" = 1 ]; then
     command -v python3 && python3 ./tests/lib/tools/mountinfo.query --run-unit-tests
     command -v python2 && python2 ./tests/lib/tools/version-compare --run-unit-tests
     command -v python3 && python3 ./tests/lib/tools/version-compare --run-unit-tests
+    command -v pytest-3 && pytest-3 ./release-tools
```

Comment:
Some packages needs to be installed for tests to work properly that are not installed by default `bs4`, `flake8`, on a fresh virtual environment I had to install `bs4`, `markdown`, `python-debian` and `flake8`.

Do you think we could/should skip the test gracefully if those modules are not installed?